threedi_scenario_downloader/downloader.py

```diff
@@ -339,11 +346,25 @@ def transform_to_list(var, length=1):
             if type(scenario) is str:
                 # assume uuid
                 scenario_instance = get_scenario_instance(scenario)
-                raster = get_raster(scenario, raster_code)
+                if raster_code == "depth-first-dtri":
```

Comment:
Die `depth-first-dtri` en `total_damage` komen meerdere keren voor in de code.
Herhaling betekent dat je ruimte hebt voor typfouten. Water beter is, is om bovenin het bestand een constante te definiëren en hier te gebruiken.

    TOTAL_DAMAGE = "total-damage"
    ....
    if raster_code == TOTAL_DAMAGE:

Reply:
Aangepast, zodat voor al de lizard raster namen nu constante worden gebruikt, die bovenaan gedefinieerd worden.